### ISSUE TITLE
Does defining an index using the fluent API have any effect when the index already exists on the database, and migrations are not being used?

### ISSUE THREAD

**Issue by smcga** [aspnet/EntityFramework.Docs#1677]
I understand that when using migrations, indexes defined in OnModelCreating will be created in the database.

I have an existing database, with indexes already set up, and was wondering if there is a benefit to defining these indexes in my DbContext? Does EFCore use these index definitions to influence how it builds queries, or does defining an existing index have no effect?

Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f85ea32c-8d23-d61d-6036-cd772e3e95d9
* Version Independent ID: 25dd8ac4-2521-5200-0ddf-63ac8c83787a
* Content: [Indexes - EF Core](https://docs.microsoft.com/en-us/ef/core/modeling/indexes#feedback)
* Content Source: [entity-framework/core/modeling/indexes.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/modeling/indexes.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @rowanmiller
* Microsoft Alias: **divega**

**Comment by smcga** [comment]
I found an answer for this on SO - https://stackoverflow.com/questions/54299018/what-is-the-benefit-of-adding-hasindex-in-your-mappings-on-a-dbfirst-scenari

tl;dr - If you are not updating the DB with migrations, this does nothing.

**Comment by offirpeer** [comment]
If I created an index in my DB, how can I tell EF to use it when fetching the data?
https://stackoverflow.com/questions/62881671/ef-core-not-fetching-data-from-table-based-on-its-clustered-index
